lib/helpers/is-object.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = object => object && typeof object === 'object' && !Array.isArray(object);
```

Comment:
Chequear si existe antes del typeof no hace falta. 

Ya que en ese caso seria `typeof object // -> "undefined"` 

Por eso propongo

```js
module.exports = object => typeof object === 'object' && !Array.isArray(object);
```

Reply:
Pero `null` es typeof `object` :fearful:

Reply:
No consideré el caso de null porque no me pareció que te pudiera llegar asi.. pero si te puede llegar bueno.. que quede..

(igual null es un object, y cumpliria con is-object)